terragrunt/env/backend.tf

```diff
@@ -0,0 +1,10 @@
+# Generated by Terragrunt. Sig: nIlQXj57tbuaRZEa
+terraform {
+  backend "s3" {
+    bucket         = "url-shortener-staging-tf"
+    dynamodb_table = "terraform-state-lock-dynamo"
+    encrypt        = true
+    key            = "./terraform.tfstate"
+    region         = "ca-central-1"
+  }
+}
```

Comment:
You can remove this file since it'll get auto created by this chunk of code in the root `terragrunt.hcl` file:
https://github.com/cds-snc/url-shortener/blob/ed714e3b14085adff5c1592f3e20d2e2e9f0131e/terragrunt/env/terragrunt.hcl#L30-L45

Reply:
Removed the file